components/script/layout_wrapper.rs

```diff
@@ -757,46 +779,48 @@ impl<'ln> ServoThreadSafeLayoutNode<'ln> {
     }
 }
 
+// NB: The implementation here is a bit tricky because elements implementing
+// pseudos are supposed to return false for is_element().
```

Comment:
Yeah, I'd like this to change. In every other browser a pseudo-element is represented as an `Element`. Of course, not in the scope of this PR.